pareto/strategic_water_management/run_strategic_model.py

```diff
@@ -227,9 +115,3 @@
     output_units=OutputUnits.user_units,
     fname="strategic_optimization_results.xlsx",
 )
-
-# This shows how to read data from PARETO reports
-set_list = []
-parameter_list = ["v_F_Trucked", "v_C_Trucked"]
-fname = "strategic_optimization_results.xlsx"
-[sets_reports, parameters_report] = get_data(fname, set_list, parameter_list)
```

Comment:
What is the reason for removing these lines?

Reply:
The reason was this read the results which caused warnings as the results files do not have standard pareto input tab names.

Reply:
Agree with Javal's comment. It was odd to have it in the run_strategic_file (mostly for developers). This example is in the documentation and in a jupyter notebook example.